default.nix

```diff
@@ -19,16 +19,16 @@ in
     python_packages = [
       "--extra-index-url"
       "https://pypi.nvidia.com"
-      "tensorrt_llm==0.8.0"
-      "torch==2.1.2"
-      "tensorrt==9.2.0.post12.dev5"
-      "tensorrt-bindings==9.2.0.post12.dev5"
-      "tensorrt-libs==9.2.0.post12.dev5"
+      "tensorrt_llm==0.9.0.dev2024040900"
+      "torch==2.2.2"
+      "tensorrt==9.3.0.post12.dev1"
+      "tensorrt-bindings==9.3.0.post12.dev1"
+      "tensorrt-libs==9.3.0.post12.dev1"
       "nvidia-pytriton==0.5.2" # corresponds to 2.42.0
       "httpx"
-      "nvidia-cublas-cu12<12.4"
-      "nvidia-cuda-nvrtc-cu12<12.4"
-      "nvidia-cuda-runtime-cu12<12.4"
+      "nvidia-cublas-cu12<12.2"
+      "nvidia-cuda-nvrtc-cu12<12.2"
```

Comment:
no chance of dropping this?

Reply:
I previously got away with this because torch==2.1.2+cu121 bundled their own cuda. Not that mixing them is a good idea.

```
$ uv pip compile - --extra-index-url https://pypi.nvidia.com -p 3.10 --prerelease=allow --annotation-style=line --extra-index-url https://download.pytorch.org/whl/cu121 --index-strategy unsafe-any-match
tensorrt_llm==0.9.0.dev2024040900
nvidia-cublas-cu12>12.2
  × No solution found when resolving dependencies:
  ╰─▶ Because torch>=2.2.0 depends on nvidia-cublas-cu12==12.1.3.1 and only the following versions of torch are available:
          torch<2.2.0a0
          torch==2.2.0
          torch==2.2.0+cu121
          torch==2.2.1
          torch==2.2.1+cu121
          torch==2.2.2
          torch==2.2.2+cu121
          torch>2.3.0a0
      we can conclude that torch>=2.2.0a0 depends on nvidia-cublas-cu12==12.1.3.1.
      And because tensorrt-llm==0.9.0.dev2024040900 depends on torch>=2.2.0a0, we can conclude that tensorrt-llm==0.9.0.dev2024040900 depends on nvidia-cublas-cu12==12.1.3.1.
      And because you require tensorrt-llm==0.9.0.dev2024040900 and you require nvidia-cublas-cu12>12.2, we can conclude that the requirements are unsatisfiable.
```